sources/common/src/TxtParser.cpp

```diff
@@ -64,38 +64,33 @@ void PARSER::TxtParser::Parse()
     std::string input_line;
     while (std::getline(m_LogFile, input_line))
     {
-        std::vector<const SIGNAL::Signal *> vpSignals =
-            m_rSignalFactory.Create(input_line, lineNumber, m_SourceHandle);
+        std::vector<const SIGNAL::Signal *> vpSignals{};
+        try
+        {
+            vpSignals = m_rSignalFactory.Create(input_line, lineNumber, m_rSignalDb->GetPrefix(), m_SourceHandle);
+        }
+        catch (const EXCEPTION::VcdException &rException)
+        {
+            if (EXCEPTION::Error::INCONSISTENT_SIGNAL == rException.GetId())
+            {
+                while (!vpSignals.empty())
+                {
+                    delete vpSignals.back();
```

Comment:
So we have unique and shared ptrs but yet you need to manually delete this here? :-) (This is known issue I think, we create signals manually using new)

Reply:
That's correct. Still new is used there.